core/chains/evm/txmgr/evm_tx_store.go

```diff
@@ -902,6 +902,12 @@ func (o *evmTxStore) FindReceiptsPendingConfirmation(ctx context.Context, blockN
 	return
 }
 
+func (o *evmTxStore) FindHighestSequence(fromAddress common.Address, chainId *big.Int) (nonce evmtypes.Nonce, err error) {
+	sql := `SELECT MAX(nonce) FROM evm.txes WHERE from_address = $1 and evm_chain_id = $2`
```

Comment:
Use parent context here as queryer?

Reply:
I think I'll need to wait on the parent context [PR](https://github.com/smartcontractkit/chainlink/pull/10735) to be merged before this one so I can update it to use the new merged context we're introducing.

Reply:
Since the other PR has been merged, I've rebased and added a parent context here now